tests/e2e_logging.py

```diff
@@ -1457,21 +1470,6 @@ def additional_interfaces(local_node_id):
         if "v8" not in args.package:
             network = test_historical_receipts(network, args)
             network = test_historical_receipts_with_claims(network, args)
-
-
-def run_parsing_errors(args):
-    txs = app.LoggingTxs("user0")
-    with infra.network.network(
-        args.nodes,
-        args.binary_dir,
-        args.debug_nodes,
-        args.perf_nodes,
-        pdb=args.pdb,
-        txs=txs,
-    ) as network:
-        network.start_and_open(args)
-
-        network.ignore_error_pattern_on_shutdown("Error parsing HTTP request")
```

Comment:
No need for a specific runner for these test cases anymore as log lines are no longer displayed on parsing error.